skymap_scanner/recos/dummy.py

```diff
@@ -30,6 +30,7 @@ class Dummy(RecoInterface):
     """Logic for a dummy reco."""
 
     def __init__(self):
+        super().__init__
```

Comment:
Maybe you're planning to do something with this, but is this a typo?

Reply:
This was accidentally commited. Will remove.